packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -335,18 +346,16 @@ module.exports = {
     ],
   },
   plugins: [
-    // Makes some environment variables available in index.html.
-    // The public URL is available as %PUBLIC_URL% in index.html, e.g.:
-    // <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    // In development, this will be an empty string.
-    new InterpolateHtmlPlugin(env.raw),
     // Generates an `index.html` file with the <script> injected.
     new HtmlWebpackPlugin({
       inject: true,
       template: paths.appHtml,
     }),
-    // Add module names to factory functions so they appear in browser profiler.
-    new webpack.NamedModulesPlugin(),
+    // Makes some environment variables available in index.html.
+    // The public URL is available as %PUBLIC_URL% in index.html, e.g.:
+    // <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    // In development, this will be an empty string.
+    new InterpolateHtmlPlugin(env.raw),
```

Comment:
ditto: why reordered?

Reply:
From the top of my head (i think i documented this somewhere) html-plugin switched to sync behaviour in the new plugin API which means it had to be instantiated first to make their own hooks available for tapping - i will double check this. thanks.

Reply:
verified:  you get `Cannot read property 'tap' of undefined` if we put interpolate plugin before html webpack plugin.